inputs/TransactionForm.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { useState } from 'react';
+import { Text, View, TextInput, TouchableOpacity, Platform } from 'react-native';
+import DateTimePicker from '@react-native-community/datetimepicker';
+import styles from '../styles';
+
+export default function TransactionForm() {
+    const [description, setDescription] = useState('');
+    const [amount, setAmount] = useState('');
+    const [date, setDate] = useState(new Date());
+    const [showDatePicker, setShowDatePicker] = useState(false);
+
+    const handleDateChange = (event: any, selectedDate: Date | undefined) => {
+        const currentDate = selectedDate || date;
+        setShowDatePicker(Platform.OS === 'ios');
+        setDate(currentDate);
+    };
+
+    const handleDatePress = () => {
+        if (Platform.OS === 'ios'){
```

Comment:
If this is true for both android and ios, is there a reason for the platform check? Wouldnt we use platform check only if there are differences between the platforms?